src/app/Common/DAL/BaseRepository.cs

```diff
@@ -114,16 +128,35 @@ private FilterDefinition<TDocument> BuildFilterQuery(TFilter filter)
                 filterQueries.Add(GetFilterById(filter.Id));
             }
 
-            if (!filterQueries.Any() && !filter.IsEmptyFilterAllowed)
-            {
-                throw new ApplicationException("Empty filter is not allowed");
-            }
-
             return filterQueries.Any()
                 ? Builders<TDocument>.Filter.And(filterQueries)
                 : FilterDefinition<TDocument>.Empty;
         }
 
+        private SortDefinition<TDocument> BuildSortQuery(IList<(string Key, SortDirection Value)> sortFields = null)
+        {
+            var defaultSortFields = new List<(string, SortDirection)>
```

Comment:
Do you think it's reasonable to add sort by default for all the queries?

I'm thinking about possible future performance degradations caused by this.

Reply:
Ok, it makes sense to remove it.